lib/file/widgets/file_tile.dart

```diff
@@ -18,24 +18,24 @@ class FileTile extends StatelessWidget {
   final void Function(File file) onOpenDirectory;
   final void Function(File file) onDownloadFile;
   void Function(File file) onTapHandler(File file) =>
-      file.isDirectory ? onOpenDirectory : onDownloadFile;
+      file == null ? null : file.isDirectory ? onOpenDirectory : onDownloadFile;
 
   @override
   Widget build(BuildContext context) {
     return EntityBuilder<File>(
       id: fileId,
-      builder: handleEdgeCases((context, file, _) {
+      builder: handleError((context, file, _) {
         final subtitle = [
-          if (file.isActualFile) file.sizeAsString,
-          if (file.updatedAt != null) file.updatedAt.shortDateTimeString,
-        ].join(', ');
+          if (file?.isActualFile == true) file.sizeAsString,
+          if (file?.updatedAt != null) file.updatedAt.shortDateTimeString,
```

Comment:
`updatedAt` may not be null, so we can simplify this expression.
```suggestion
          if (file != null) file.updatedAt.shortDateTimeString,
```